nidirect_common/inc/form_alter.inc

```diff
@@ -127,8 +127,8 @@ function nidirect_common_form_node_form_alter(&$form, FormStateInterface $form_s
   // Hide 'promote to front-page' option.
   $form['promote']['#access'] = FALSE;
 
-  // Hide 'Sticky at top of lists' option, unless it's a news node.
-  if (!preg_match('/^node_news/', $form['#form_id'])) {
+  // Hide 'Sticky at top of lists' option, except for news and contact nodes.
+  if (!preg_match('/^node_news/', $form['#form_id']) && !preg_match('/^node_contact/', $form['#form_id'])) {
```

Comment:
For a little more conciseness, how about `if (!preg_match('/^node_(news|contact)/', $form['#form_id'])) {` ?

Reply:
Yes - nice one